spacy/pipeline/transition_parser.pyx

```diff
@@ -312,12 +332,15 @@ class Parser(TrainablePipe):
         del backprop_scores
         return losses
 
-    def get_loss(self, states_scores, examples):
-        states, scores = states_scores
+    def get_loss(self, states_scores, examples, max_moves):
```

Comment:
Missing type hints.

Reply:
I think we should do these in a separate PR, practically nothing in these modules have type hints. So I'd be more in favor of adding them all at once in a separate PR.

Reply:
I’ll create a ticket for it tomorrow.

Reply:
Created a ticket. I resolved the other type hints comments.